### PR TITLE
Add type definitions for csp-hash-generator

### PR DESCRIPTION
**Pull request by bobcode99**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by bobcode99** [comment]
@miguel-a-calles-mba 
Hi, i add type defenitions to your  [csp-hash-generator](https://github.com/miguel-a-calles-mba/csp-hash-generator) library.

If can, please also help me review.

Thanks!

File: types/csp-hash-generator/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Type definitions for csp-hash-generator 0.0
+// Project: https://github.com/miguel-a-calles-mba/csp-hash-generator
+// Definitions by: yb999 <https://github.com/bobcode99>
```
Comment (peterblazejewicz) [inline types/csp-hash-generator/index.d.ts:3]: the handle name is different

Comment (bobcode99) [reply types/csp-hash-generator/index.d.ts]: Resolve https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58369/commits/050b96854a63859dafd7252f4a228ba0376da7a4

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by bobcode99** [comment]
Ready to merge

**Comment by miguel-a-calles-mba** [comment]
Awesome. Thanks @bobcode99!

**Comment by bobcode99** [comment]
> Awesome. Thanks @bobcode99!

You re welcome.

**Pull request merged**